packages/fluentui/react-northstar/test/specs/components/Form/FormField-test.tsx

```diff
@@ -23,6 +23,15 @@ describe('FormField', () => {
   formFieldImplementsShorthandProp('message', Text);
   formFieldImplementsShorthandProp('control', Box, { mapsValueToProp: 'children' });
 
+  let testContainerInfo: ReturnType<typeof createTestContainer> | undefined;
+
+  afterEach(() => {
+    if (testContainerInfo) {
```

Comment:
**suggestion:** using globals in test is a bad practice. "sometimes" it is justifiable but not in this example. 

why? test container is used only in one place. can you collocate it there or at least encapsulate the logic in a `setup` function. 

Test code should be approached like implementation - readable and clean

Reply:
If it's created as a local, it can't be easily cleaned up if the test fails. Also I'd like to go with a minimal solution for now and like @layershifter mentioned they can make general improvements later.